invenio_communities/views/decorators.py

```diff
@@ -66,3 +65,25 @@ def view(**kwargs):
             raise PermissionDeniedError()
 
     return view
+
+
+def pass_community_endpoint(f):
+    """Decorate a view to calculate community endpoint."""
+    @wraps(f)
+    def view(**kwargs):
+        """."""
+        community = kwargs['community']
+        base_endpoint = f'/api/communities/{community.to_dict()["uuid"]}/members'  # TODO: change to community.id (once backend ready)
```

Comment:
To be discussed what is the expected future, use everywhere human-readable ID or use everywhere the UUID for a community.